src/sass/misc/_panel.scss

```diff
@@ -119,11 +119,11 @@
         .panel-title { color: $accentColor; }
 
         .btn {
-            border-color: #7dc5f2;
-            color: #7dc5f2;
+            border-color: #7dc6f8;
```

Comment:
It's not immediately obvious without reference to this issue why this is here, and why it was changed.

I might add a comment to say that this is basically `$whicheverVariable` but with contrast adjusted to meet WCAG thresholds.

Reply:
comment added

Reply:
The lint output shows you the expected formatting, which you'll have to fix.

(Sorry that it seems overzealous about a comment!)

Reply:
The PR is already merged, is there anything more to be fixed?

Reply:
nope, i was only explaining the problem. (And I only saw the first commit...)

Normally we would cherry pick website fixes to the current branch but right now it's due to be released ~soon. @adilmp1 I would check in with @ericvn or on #release-1.21 on slack as to if this needs to be cherrypicked to the release-1.21 branch or if that hasn't happened yet.

(If you don't understand any of those terms, please ask there, or Faseela or myself.)